### ISSUE TITLE
Feature request: add SURF_WIND derived variable (vector of UBOT, VBOT)

### ISSUE THREAD

**Issue by susburrows** [UV-CDAT/uvcmetrics#111]

**Comment by painter1** [comment]
Is this for a vector-contour plot, aka plot set 6?  Probably this should wait until I have a chance to redo the data structure.  The present one (just a tuple) doesn't work well with everything else.

**Comment by susburrows** [comment]
Hi Jeff, yes, I think a vector-contour plot would be appropriate.

Also, I should say that this is really a low priority, and I think there are probably much better approaches available than what can be quickly done in the present code.  I really just wanted to have this on the list before I forget; it is something we might consider doing later if it becomes more important or if we have extra time, or if it becomes easier after other developments.

**Comment by painter1** [comment]
It would be useful to have a general-purpose vector plot capability.  That would cover (UBOT,VBOT) as well as what Jerry Potter just tried today: a vertical average of the horizontal wind (U,V = zonal,meridional velocity). And it would cover whatever else comes up next.

**Comment by doutriaux1** [comment]
You mean contour with arrows on it? Kind of like streamlines? VCS use to be able to do this. Do you have an example plot you can post here?

**Comment by painter1** [comment]
Yes, a vector-contour plot is vectors (little arrows) overlaying a contour plot (isofill).  We're doing that now, with VCS.  Thus the issue isn't "how to make a plot".  The issue is mainly about setting up the input system, and probably a bunch of minor interior tweaks.  There might be some difficulties in getting the arrows to be the right size and density, but more likely not.
